src/api/neighborhood-finder/get-address-meta.js

```diff
@@ -27,6 +27,12 @@ exports.addressHandler = async (req, res, next) => {
       },
       timeout: 1000, // milliseconds
     });
+    if (geoResult.data.status == "ZERO_RESULTS") {
```

Comment:
huh cool! good to know there is a specific status from the API to indicate empty results! nice find.

i think you have a linting error (see below) 👇